test/test_getting_columns.py

```diff
@@ -285,3 +283,45 @@ def test_columns_with_keywords():
         "drop_20",
         "create_table",
     ]
+
+
+def test_columns_with_aliases_same_as_columns():
```

Comment:
Great for adding test cases here 🙂 

However, let's try to add here the smallest possible queries. In this case what about the following:

```sql
    select sum(impressions) as impr, sum(clicks) as clicks from amazon_pl.search_term_report_impala
```

Reply:
Since you are writing it's probably not working🤣 the columns should be `impressions` and `clicks` (which is both column name and alias, but since it's self alias it's fine)

Reply:
Nope I checked it's working fine, added more tests and a fix for doubles in resolved aliases if alias refers to list of columns